WordPress/Classes/Models/WPAccount.m

```diff
@@ -126,4 +126,11 @@ - (WordPressComApi *)restApi
     return _restApi;
 }
 
+#pragma mark - WordPress.com support methods
+
+- (BOOL)isWPComAccount
+{
+    return _restApi != nil;
```

Comment:
Note here: If the `restApi` getter isn't called before `isWPComAccount`, `_restApi` ivar will be nil, and thus, this method may return *false*, even when we're dealing with a Dotcom account.

Second issue, which wasn't introduced by this PR, is that if you just call the `restApi` getter, it'll return a `WordPressComApi` instance, even if the authToken is nil. Should we add a check for that?